packages/test-data/pdrs/cross-collection-id-collision.PDR

```diff
@@ -0,0 +1,39 @@
+ORIGINATING_SYSTEM = MODAPS_CUMULUS_FPROC;
+TOTAL_FILE_COUNT = 4;
+EXPIRATION_TIME = 2017-08-22T20:07:21;
+OBJECT=FILE_GROUP;
+ DATA_TYPE = MOD09GQ;
+ DATA_VERSION = 006;
+ OBJECT = FILE_SPEC;
+  DIRECTORY_ID = /MODOPS/MODAPS/EDC/CUMULUS/FPROC/DATA;
+  FILE_ID = MOD09GQ.A2017224.h09v02.006.2017227165020.hdf;
+  FILE_TYPE = HDF;
+  FILE_SIZE = 17865615;
+  FILE_CKSUM_TYPE = CKSUM;
+  FILE_CKSUM_VALUE = 4208254019;
+ END_OBJECT = FILE_SPEC;
+ OBJECT = FILE_SPEC;
+  DIRECTORY_ID = /MODOPS/MODAPS/EDC/CUMULUS/FPROC/DATA;
+  FILE_ID = MOD09GQ.A2017224.h09v02.006.2017227165020.hdf.met;
+  FILE_TYPE = METADATA;
+  FILE_SIZE = 44118;
+ END_OBJECT = FILE_SPEC;
+END_OBJECT = FILE_GROUP;
+OBJECT=FILE_GROUP;
+ DATA_TYPE = MOD87GQ;
+ DATA_VERSION = FAKE;
+ OBJECT = FILE_SPEC;
+  DIRECTORY_ID = /MODOPS/MODAPS/EDC/CUMULUS/FPROC/DATA;
+  FILE_ID = MOD09GQ.A2017224.h09v02.006.2017227165020.hdf;
+  FILE_TYPE = HDF;
+  FILE_SIZE = 17865615;
+  FILE_CKSUM_TYPE = CKSUM;
+  FILE_CKSUM_VALUE = 4208254019;
+ END_OBJECT = FILE_SPEC;
+ OBJECT = FILE_SPEC;
+  DIRECTORY_ID = /MODOPS/MODAPS/EDC/CUMULUS/FPROC/DATA;
+  FILE_ID = MOD09GQ.A2017224.h09v02.006.2017227165020.hdf.met;
+  FILE_TYPE = METADATA;
+  FILE_SIZE = 44118;
+ END_OBJECT = FILE_SPEC;
+END_OBJECT = FILE_GROUP;
```

Comment:
We can also use this PDR in the integration test to verify that granules with same producerGranuleId from different collections are ingested into the system.

Reply:
I originally read this as 'this will be a benefit when we write the cross-collection integration test', not 'we should do that here'.     I'm doing a bit of work this morning to see how complicated this modification to the existing integration test is, given writing the test was understood  scope creep  given the test plan/implementation work progress.

Reply:
Talked with @jennyhliu , we're going to address the IART ticket constraint in the integration test tickets.